test/missiles_test.cpp

```diff
@@ -11,7 +11,7 @@ using ::testing::Lt;
 using ::testing::Pair;
 using ::testing::UnorderedElementsAre;
 
-void TestArrowRotatesUniformly(Missile &missile, int startingFrame, int leftFrame, int rightFrame)
+void TestArrowRotatesUniformly(Missile &missile, int startingFrame, unsigned leftFrame, unsigned rightFrame)

```

Comment:
leftFrame/rightFrame are meant to be compared to missile._miAnimFrame (set from startingFrame), they should all be the same type. Assuming this was an accidental include when only the counter half of the pair was updated to use unsigned constants?

Reply:
I changed the parameter because they get compared to the ovbserved values.
But you are right, then I should also update `TestAnimatedMissileRotatesUniformly`.

From my observation this shouldn't be a problem for the tests, because `_miAnimFrame` is always positive in the tests. But the tests should also be strict. 🤔 

Feel free to make a PR 🙂